packages/space-opera/package.json

```diff
@@ -15,7 +15,7 @@
     "watch:test": "karma start",
     "watch:tsc": "tsc -w",
     "watch:rollup": "rollup -c -w",
-    "serve": "./node_modules/.bin/http-server -a 127.0.0.1 -o /demo/ -c-1",
+    "serve": "./node_modules/.bin/http-server -a 127.0.0.1 -o /editor/ -c-1",
```

Comment:
What does this change do?

Reply:
Sure, that sounds good. I don't generally run `dev`, which is probably why I haven't noticed.